plugins/plugin-svn/che-plugin-svn-ext-subversion/src/main/java/org/eclipse/che/ide/ext/svn/client/common/filteredtree/ProjectTreeNodeDataAdapter.java

```diff
@@ -0,0 +1,100 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.ext.svn.client.common.filteredtree;
+
+import org.eclipse.che.ide.api.project.tree.AbstractTreeNode;
+import org.eclipse.che.ide.api.project.tree.TreeNode;
+import org.eclipse.che.ide.ui.tree.NodeDataAdapter;
+import org.eclipse.che.ide.ui.tree.TreeNodeElement;
+
+import java.util.List;
+
+/**
+ * An {@link NodeDataAdapter} that allows to visit the {@link AbstractTreeNode}.
+ *
+ * TODO temporary moved here, will be removed after resolving IDEX-3045
+ *
```

Comment:
It's not solved for now and is part of the [epic](https://jira.codenvycorp.com/browse/CHE-255).